src/credentials/activation.ts

```diff
@@ -40,36 +42,49 @@ export async function loginWithMostRecentCredentials(
     const profileNames = Object.keys(providerMap)
     const previousCredentialsId = toolkitSettings.readSetting<string>(profileSettingKey, '')
 
+    async function tryConnect(creds: CredentialsProviderId, popup: boolean): Promise<boolean> {
```

Comment:
Does this function need to be scoped to loginWithMostRecentCredentials?

Reply:
Is there a reason not to?

Reply:
No particular reason aside from that we typically don't.